src/index.ts

```diff
@@ -86,6 +88,9 @@ async function accessCards(
   try {
     const { data: accessCards } = await client.get<AccessCard[]>(
       `${config.baseUrl}/user/users/${userId}/accessCards?apiKey=${config.apiKey}`,
+      {
+        headers: { Cookie: `${config.authCookie}=${code}'` }
```

Comment:
If code is empty, I wouldn't touch the cookie. Here as I see, we clear it out.

Reply:
authCookie="" will work as undefined value.

Reply:
Will undefined code erase cookie?
In other words, is here any difference with

```
if (code === undefined) {
// request without headers
}
else {
// headers: { Cookie: `${config.authCookie}=${code}'` }
}
```
?

Reply:
Updated: send headers with cookie when code defined only